src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import java.sql.Timestamp
+import java.util.UUID
+import org.joda.time.DateTime
+
+/**
+ * Created by mbemis on 2/18/16.
+ */
+
+case class SubmissionRecord(id: UUID,
+                            submissionDate: Timestamp,
+                            submitterId: String,
+                            methodConfigurationId: Long,
+                            submissionEntityId: Option[Long],
+                            status: Long
+                             )
+
+case class SubmissionStatusRecord(id: Long, name: String)
+
+case class SubmissionWorkflowRecord(submissionId: UUID, workflowId: UUID)
+
+case class SubmissionWorkflowFailureRecord(submissionId: UUID, workflowFailureId: Long)
+
+case class SubmissionValidationRecord(workflowId: UUID, value: Option[Long], errorText: Option[String], inputName: String)
```

Comment:
Per the domain object transition doc:

SUBMISSION_VALIDATION_VALUE    // to case class
    workflow: FK to WORKFLOW.id / WORKFLOW_FAILURE.id (does this work?)
     value: FK to ATTRIBUTE.id (nullable)
     error: String (nullable)
     inputName: String

What should the approach be here? I've included just workflowId for the sake of having something that compiles. Should it be something like: workflowId: Option[UUID], workflowFailureId: Option[Long] ?